back-end/aws-cloud9-api/src/main/java/com/serverless/dal/ClosedQuestion.java

```diff
@@ -0,0 +1,94 @@
+package com.serverless.dal;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGeneratedKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey;
+
+public class ClosedQuestion extends Question {
+	
+		private String id;
+		private String name;
+		private String question;
+		private String form_membership;
```

Comment:
Jeśli ClosedQuestion dziedziczy po Question to po co to pisać? Wystarczy w klasie bazowej ustawić to jako protected.